webpack.config.js

```diff
@@ -18,12 +18,9 @@ module.exports = {
     },
     module: {
         loaders: [{
-            test: /(\.jsx?)$/,
+            test: /\.js/,
             exclude: /(bower_components|node_modules)/,
-            loader: 'babel',
-            query: {
-                cacheDirectory: true
-            }
+            loader: 'imports?this=>window'
```

Comment:
I can also fix this in the follow up PR.  Should be an easy fix just a lot of files to change and I didn't want to pollute the diff.